### PR TITLE
[TRIVIAL] fix typo in SECURITY.md

### PR DESCRIPTION
**Pull request by sokkaofthewatertribe**
noticed a minor typo while reviewing security.md

### CODE REVIEW

**Review by ximinez — APPROVED** [review]
Nice catch. (I've changed the title of this PR to tag it as "trivial", which allows it to be merged with only one approval.)

**Comment by sokkaofthewatertribe** [comment]
No problem, thanks @ximinez

**Comment by sokkaofthewatertribe** [comment]
Hey @intelliot -  I noticed that merging this PR is blocked because the initial commit was not signed. Should I open a new PR with a signed commit, or are you still able to merge?

File: SECURITY.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Your report should include the following:
 - The steps to reproduce the vulnerability;
 - Any other relevant details or artifacts, including code, scripts or patches.
 
-In your mail, please describe of the issue or the potential threat; if possible, please include a "repro" (code that can reproduce the issue) or describe the best way to reproduce and replicate the issue. Please make your report as extensive as possible.
+In your mail, please describe the issue or the potential threat; if possible, please include a "repro" (code that can reproduce the issue) or describe the best way to reproduce and replicate the issue. Please make your report as extensive as possible.
```
Comment (intelliot) [inline SECURITY.md:40]:
```suggestion
In your email, please describe the issue or potential threat. If possible, include a "repro" (code that can reproduce the issue) or describe the best way to reproduce and replicate the issue. Please make your report as detailed and comprehensive as possible.
```

**Comment by intelliot** [comment]
> Should I open a new PR with a signed commit, or are you still able to merge?

There's no need to open a new PR to sign the commits; you can sign and force-push to your existing branch, which will update this PR.

**Comment by intelliot** [comment]
Merging because [TRIVIAL]

**Pull request merged**